weddineo-frontend/libs/rest-api/src/lib/variables.ts

```diff
@@ -1,3 +1,3 @@
 export const variables = {
-  API_URL: 'http://localhost/'
+  API_URL: 'http://localhost:8080/'
```

Comment:
I think API URL should be localhost:8080/api. See application.properties on backend project.

Reply:
Done